.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,3 +8,22 @@ Follow this checklist to help us incorporate your contribution quickly and easil
  - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).
 
 It will help us expedite review of your Pull Request if you tag someone (e.g. `@username`) to look at it.
+
+Tests status
```

Comment:
I chose "health of master" to make it very clear that this indicates nothing about the PR, but just about where it will be merged. I am not arguing that you have to say exactly my words, but I think "Tests status" could be confusing.